<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册框</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>注册新用户</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>72</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editPass">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="editNickName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>12</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editConfirmPass">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxSex">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>78</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>男</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>女</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>其他</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>昵称</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbQuit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string> 确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>41</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editServerIP">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>38</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>editName</tabstop>
  <tabstop>editPass</tabstop>
  <tabstop>editConfirmPass</tabstop>
  <tabstop>editNickName</tabstop>
  <tabstop>cbxSex</tabstop>
  <tabstop>pbOk</tabstop>
  <tabstop>pbQuit</tabstop>
  <tabstop>editServerIP</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pbQuit</sender>
   <signal>clicked()</signal>
   <receiver>RegisterDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
